src/sinopiaServer.js

```diff
@@ -3,10 +3,15 @@
 import SinopiaServer from 'sinopia_server'
 import CognitoUtils from './CognitoUtils'
 import Config from './Config'
+<<<<<<< HEAD
 import {
   rtFixturesGroups, getFixtureResourceTemplate,
   listFixtureResourcesInGroupContainer, resourceTemplateIds,
 } from '../__tests__/fixtureLoaderHelper'
+=======
+// eslint-disable-next-line node/no-unpublished-import
+import { rtFixturesGroups, getFixtureResourceTemplate, listFixtureResourcesInGroupContainer } from '../__tests__/fixtureLoaderHelper'
+>>>>>>> 6f1c60fd... Eslint fixes
```

Comment:
This shouldn't be here.

Reply:
Forgot to hit save when fixing merge ...